p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -1567,14 +1567,17 @@ directions:
   `out`, or `inout`); this applies to `package`, `control`, `parser`,
   and `extern` objects. Values for these parameters must be specified
   at compile-time, and must evaluate to compile-time known values.
+  See Section [#sec-parameterization] for further details.
 - For actions all directionless parameters must be at the end of the
   parameter list.  When an action appears in a `table`'s `actions`
   list, only the parameters with a direction must be bound.
+  See Section #sec-actions for further details.
```

Comment:
I don't know all Madoko markup syntax well enough but should this #sec-actions be enclosed in square brackets like the other nearby cross-refs are?